typescript/packages/well-completions-plot/src/types/dataTypes.ts

```diff
@@ -0,0 +1,93 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+export type AttributeType = string | number | boolean | undefined;
+export enum SortDirection {
+    Ascending = "Ascending",
+    Descending = "Descending",
+}
+
+export enum SortBy {
+    Name = "well name",
+    StratigraphyDepth = "stratigraphy depth",
+    CompletionDate = "earliest comp date",
+}
+
+export const SortByEnumToStringMapping = {
+    [SortBy.Name]: "well name",
+    [SortBy.StratigraphyDepth]: "stratigraphy depth",
+    [SortBy.CompletionDate]: "earliest comp date",
```

Comment:
Use capital letters? These should only be used when exposing enum to GUI, hence, this is what the user is going to see when interacting with the enum.